src/prefect/settings/models/experiments.py

```diff
@@ -1,11 +1,33 @@
-from typing import ClassVar
+from __future__ import annotations
+
+from typing import ClassVar, Optional
 
 from pydantic import AliasChoices, AliasPath, Field
 from pydantic_settings import SettingsConfigDict
 
 from prefect.settings.base import PrefectBaseSettings, build_settings_config
 
 
+class _LogfireSettings(PrefectBaseSettings):
+    """
+    Settings for Logfire tracing
+    """
+
+    model_config: ClassVar[SettingsConfigDict] = build_settings_config(
+        ("experiments", "logfire")
+    )
+
+    enabled: bool = Field(
+        default=False,
+        description="If `True`, enables Logfire tracing. Set to `False` to disable Logfire tracing.",
+    )
+
+    write_token: Optional[str] = Field(
+        default=None,
+        description="The Logfire write token to use for tracing.",
+    )
```

Comment:
If `logfire` isn't going to be installed via normal operation, I don't think we should include these settings as part of our schema. If we want to keep these settings then I think we should add a `logfire` extra. Otherwise, I think we can use plain environment variables to control `logfire`.